python_ta/debug/accumulation_table.py

```diff
@@ -0,0 +1,158 @@
+"""
+Table data structure that prints a nicely formatted table
+for an accumulator loop
+"""
+from __future__ import annotations
+
+import inspect
+import sys
+
+import astroid
+import tabulate
+
+
+def no_of_whitespaces(with_line: str) -> int:
+    """Return the number of whitespaces that are in the line of code
```

Comment:
This description should be more generic, and not assume that the line of code contains a with statement (or in fact is a line of code at all). Something like "number of whitespace at the start of <param name>" is better. Also choose a better parameter name.

And finally, all docstring statements should end with a period (make sure to do this throughout).